hello-world-sdk/README.md

```diff
@@ -0,0 +1,172 @@
+This quickstart requires you to have the following installed on your machine:
+- [Docker](https://docs.docker.com/)
+- [Go](https://golang.org/)
+- [Postman](https://www.getpostman.com/) [Optional]
+
+## Step 1 - Setup Dapr
+
+Follow [instructions](https://docs.dapr.io/getting-started/install-dapr/) to download and install the Dapr CLI and initialize Dapr.
+
+## Step 2 - Understand the code
+
+Now that Dapr is set up locally, clone the repo, then navigate to the Hello World SDK quickstart:
+
+```sh
+git clone [-b <dapr_version_tag>] https://github.com/dapr/quickstarts.git
+cd quickstarts/hello-world-sdk
+```
+
+> **Note**: See https://github.com/dapr/quickstarts#supported-dapr-runtime-version for supported tags. Use `git clone https://github.com/dapr/quickstarts.git` when using the edge version of dapr runtime.
+
+The [order.go](./order.go) is a simple command line application, that implements four commands:
+* `put` sends an order with configurable order ID.
+* `get` return the current order number.
+* `del` deletes the order.
+* `seq` streams a sequence of orders with incrementing order IDs.
+
+First, the app instantiates Dapr client:
+
+```go
+    client, err := dapr.NewClientWithPort(port)
+    if err != nil {
+        panic(err)
+    }
+    defer client.Close()
+```
+
+Then, depending on the command line argument, the app invokes corresponding method:
+
+Persist the state:
+```go
+    err := client.SaveState(ctx, stateStoreName, "order", []byte(strconv.Itoa(orderID)))
+```
+Retrieve the state:
+```go
+    item, err := client.GetState(ctx, stateStoreName, "order")
+```
+Delete the state:
+```go
+    err := client.DeleteState(ctx, stateStoreName, "order")
+```
+
+## Step 3 - Run the app with Dapr
+
+<!-- STEP
+expected_stdout_lines:
+  - "go mod tidy"
+  - "go mod vendor"
+  - "go build -o order order.go"
+expected_stderr_lines:
+name: "make"
+-->
+
+1. Build the app
+    ```bash
+    make
+    ```
+
+<!-- END_STEP -->
+
+2. Run the app
+
+There are two ways to launch Dapr applications. You can pass the app executable to the Dapr runtime:
+
+<!-- STEP
+expected_stdout_lines:
+  - "You're up and running! Both Dapr and your app logs will appear here."
+  - "== APP == dapr client initializing for: 127.0.0.1:3500"
+  - "== APP == Sending order ID 20"
+  - "== APP == Successfully persisted state"
+expected_stderr_lines:
+output_match_mode: substring
+name: "order put"
+background: true
+sleep: 5
+-->
+
+    ```bash
+    dapr run --app-id order-app --dapr-grpc-port 3500 -- ./order put --id 20
+    ```
+
+<!-- END_STEP -->
+
+<!-- STEP
+expected_stdout_lines:
+  - "You're up and running! Both Dapr and your app logs will appear here."
+  - "== APP == dapr client initializing for: 127.0.0.1:3500"
+  - "== APP == Getting order"
+  - "== APP == Order ID 20"
+expected_stderr_lines:
+output_match_mode: substring
+name: "order get"
+background: true
+sleep: 5
+-->
+
+    ```bash
+    dapr run --app-id order-app --dapr-grpc-port 3500 ./order get
+    ```
```

Comment:
For commands that have outputs in the quickstarts I think it's good to have an example of what it should look like. We anyone who is going through these to feel confident/be able to verify what they're seeing.